weather_mv/loader_pipeline/sinks_test.py

```diff
@@ -84,7 +84,7 @@ def setUp(self) -> None:
         super().setUp()
         self.test_data_path = os.path.join(self.test_data_folder, 'test_data_20180101.nc')
         self.test_grib_path = os.path.join(self.test_data_folder, 'test_data_grib_single_timestep')
-        self.test_tif_path = os.path.join(self.test_data_folder, 'test_data_tif_start_time.tif')
```

Comment:
Should we remove this test file: `test_data_tif_start_time.tif` ... is it required now ? Because I see you have replaced this with `test_data_tif_time.tif` at many places.

Reply:
yes, the file will no longer be needed. I will remove it.